### PR TITLE
Add sig-rktnetes to the list of SIGs

### PR DESCRIPTION
**Pull request by tmrts**

### CODE REVIEW

**Comment by tmrts** [comment]
cc @kubernetes/sig-rktnetes

**Comment by yifan-gu** [comment]
LGTM

File: sig-rktnetes/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[Meeting Notes](https://docs.google.com/document/d/1otDQ2LSubtBUaDfdM8ZcSdWkqRHup4Hqt1VX1jSxh6A/edit?usp=sharing)
```
Comment (yifan-gu) [inline sig-rktnetes/README.md]: Can we also put what's rktnetes here? What's the mission of the sig?

Comment (pskrzyns) [inline sig-rktnetes/README.md]: Meeting notes should be available for anyone. At this moment i don't have permissions

Comment (tmrts) [inline sig-rktnetes/README.md]: @yifan-gu done PTAL

@pskrzyns the new link should work PTAL

File: sig-rktnetes/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# The Mission
+`sig-rktnetes` is focused on the topics related to Kubernetes integration of container runtime rkt which is referred to as rktnetes.
```
Comment (euank) [inline sig-rktnetes/README.md]: nit: 'is focused on topics related to the integration between Kubernetes and the rkt container runtime, more succinctly referred to as "rktnetes".'

File: sig-rktnetes/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+`sig-rktnetes` is focused on the topics related to Kubernetes integration of container runtime rkt which is referred to as rktnetes. 
+
+This includes:
+- Kubernetes and rkt integration
+- Performance and scalability
+- Images, package management
+- Rktnetes deployments
+- Issues related to monitoring (cAdvsor, Heapster, etc.)
+
+The main objective is to maintain and develop the integration of rkt to the Kubernetes as the container runtime.
```
Comment (euank) [inline sig-rktnetes/README.md]: I think we can make this a little more ambitious and/or clear.

"The main objective of this SIG is to ensure that rkt is the best runtime to use with a Kubernetes cluster" perhaps?

File: sig-rktnetes/README.md (hunk 2 above)
Comment (tmrts) [inline sig-rktnetes/README.md]: ack

File: sig-rktnetes/README.md (hunk 3 above)
Comment (tmrts) [inline sig-rktnetes/README.md]: SGTM

**Comment by tmrts** [comment]
PTAL

**Comment by yifan-gu** [comment]
LGTM

File: sig-rktnetes/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# The Mission
+`sig-rktnetes` is focused on topics related to the integration between Kubernetes and the rkt container runtime, more succinctly referred to as "rktnetes"
```
Comment (euank) [inline sig-rktnetes/README.md]: tiny nit, punctuation

**Comment by euank** [comment]
one nit, LGTM

File: sig-rktnetes/README.md (hunk 4 above)
Comment (tmrts) [inline sig-rktnetes/README.md]: fixed

**Pull request merged**